src/OverviewView.js

```diff
@@ -0,0 +1,141 @@
+import { OrthographicView } from '@deck.gl/core';
+import VivView from './VivView';
+import { OverviewLayer } from './layers';
+
+/**
+ * This class generates a OverviewLayer and a view for use in the VivViewer as an overview to a Detailview (they must be used in conjection)
+ * @param {Object} viewState The viewState object.
+ * @param {Object} loader The loader, used for inferring zoom level.
+ * @param {number} detailHeight The height of the detail view.
+ * @param {number} detailWidth The width of the detail view.
+ * @param {number} overviewScale The scale of this viewport relative to the detail.
+ * @param {number} margin The margin to be offset from the the corner of the other viewport.
+ * @param {string} overviewLocation The location of the viewport - one of "bottom-right", "top-right", "top-left", "bottom-left."
```

Comment:
For this class, I'd recommend removing `overview` from the property names since they are redundant of the class name. (`overviewScale` --> `scale`, etc). I'm also somewhat inclined to use `position` rather than `location` (I think I've seen this used elsewhere but I'm actually not sure)...

Reply:
Ok.